src/applications/gi-sandbox/containers/SearchPage.jsx

```diff
@@ -53,6 +70,24 @@ export function SearchPage({
               </va-alert>
             </div>
           )}
+          {!error && !smallScreen && tabbedResults[tab]}
+          {smallScreen && (
+            <div>
+              <va-accordion>
+                <va-accordion-item header="Search by name">
+                  <NameSearchForm />
+                </va-accordion-item>
+                <va-accordion-item header="Search by location">
+                  <LocationSearchForm />
+                </va-accordion-item>
+              </va-accordion>
+              {search.name.count && (
```

Comment:
Why is this not inside the "Search by name" accordion?
Where are the filter containers for the "Search by name" accordion?

Reply:
I think that the Search by name accordion only contains the text field and button to execute the search for mobile view 
![Screen Shot 2021-07-22 at 12 12 19 PM](https://user-images.githubusercontent.com/48804834/126672571-7b2222d8-1945-4822-923f-421b8d1f20a4.png)


I was chatting with Devin about this, this story does not actually include the update to the “Update tuition and housing estimates” and “Filter your results” components.

Reply:
You don't update them, just include the components so that order is correct for the results and let the story that is for updating them handle changes.

If you use `NameSearchResults` container they will already be present and can just update how Search Result cards are displayed

Reply:
That is what I am planning to do here since we want to re-use the `NameSearchResults` and then just conditionally display it's contents for mobile. I wasn't aware that several of our stories overlapped in terms of implementation, but noticed that today after digging into the requirements a little deeper.